src/registry.ts

```diff
@@ -5,15 +5,22 @@ import { addUrlQuery, checkUint64, defaultOptions, hexToUint8Array, makeUrl, toH
 import { Buffer } from "buffer";
 import { hashDataKey, hashRegistryEntry, Signature } from "./crypto";
 
+/**
+ * Custom get entry options.
+ *
+ * @property [timeout=5] - The custom timeout for getting an entry, in seconds. The maximum value allowed is 300.
+ */
 const defaultGetEntryOptions = {
   ...defaultOptions("/skynet/registry"),
-  timeout: 5_000,
+  timeout: 5,
 };
 
 const defaultSetEntryOptions = {
   ...defaultOptions("/skynet/registry"),
 };
 
+export const MAX_GET_ENTRY_TIMEOUT = 300; // 5 minutes
```

Comment:
In `siad` we valuate whether the timeout is between 1 and 300s, we might want to do the same here.